RecoTracker/TkHitPairs/plugins/HitPairEDProducer.cc

```diff
@@ -272,6 +271,10 @@ HitPairEDProducer::HitPairEDProducer(const edm::ParameterSet& iConfig):
   else
     throw cms::Exception("Configuration") << "HitPairEDProducer requires either produceIntermediateHitDoublets or produceSeedingHitSets to be True. If neither are needed, just remove this module from your sequence/path as it doesn't do anything useful";
 
+  auto clusterCheckTag = iConfig.getParameter<edm::InputTag>("clusterCheck");
+  if(clusterCheckTag.label() != "")
+    clusterCheckToken_ = consumes<bool>(clusterCheckTag);
```

Comment:
isn't this technically equivalent to what it was before? 
That is, if the label is empty, you get an uninitialized token.